contracts/Controller.sol

```diff
@@ -838,6 +977,38 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         return whitelist.isWhitelistedCallee(_callee);
     }
 
+    /**
+     * @notice check if a vault is liquidatable in a specific round id
+     * @param _owner vault owner address
+     * @param _vaultId vault id to check
+     * @param _roundId chainlink round id to check vault status at
+     * @return vault struct, isLiquidatable, true if vault is undercollateralized, the price of 1 repaid otoken and the otoken collateral dust amount
```

Comment:
also explain that price is scaled at 1e18. 
Also why 1e18?

Reply:
I assume it's 1e18 because you hardcoded a .div(1e18) in the collateral amount calculation

Reply:
what is scaled by 1e8 ? the short amount is in 8decimals, that's why I divided by 1e8 in the end.

Reply:
The liquidation price is always in collateral short otoken collateral decimals.